src/data/monitor.json

```diff
@@ -1,378 +1,346 @@
     },
     "immersive-web/navigation": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "immersive-web/performance-improvements": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "immersive-web/real-world-geometry": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "immersive-web/spatial-favicons": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "privacycg/is-logged-in": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "privacycg/js-membranes": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "WICG/canvas-formatted-text": {
-      "lastreviewed": "2021-07-01",
-      "comment": "no published content yet"
-    },
-    "WICG/urlpattern": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "immersive-web/detached-elements": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "immersive-web/occlusion": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "WICG/handwriting-recognition": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "WICG/manifest-incubations": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "Chromium-specific"
     },
     "immersive-web/marker-tracking": {
-      "lastreviewed": "2021-07-01",
-      "comment": "no published content yet"
+      "lastreviewed": "2021-09-01",
+      "comment": "marked as unstable API"
     },
     "WICG/WebApiDevice": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet, but likely soon at https://wicg.github.io/WebApiDevice/device_attributes from https://github.com/WICG/WebApiDevice/blob/main/device_attributes/index.bs "
     },
     "immersive-web/WebXR-WebGPU-Binding": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no published content yet"
     },
     "WebAudio/web-audio-api-v2": {
-      "lastreviewed": "2021-07-01",
+      "lastreviewed": "2021-09-01",
       "comment": "no spec yet"
     },
-    "WICG/app-history": {
-      "lastreviewed": "2021-07-01",
-      "comment": "No clear implementation plan - tracked at https://www.chromestatus.com/features/6232287446302720"
-    },
-    "WICG/scheduling-apis": {
```

Comment:
This one seems to have been lost in the process - given it's implemented in Chrome 94+,  I believe it should be moved to specs.json too

Reply:
It got lost... but last time :)

The Prioritized Task Scheduling spec was added to `specs.json` last time we reviewed the list. We just forgot to drop it from `monitor.json` at the time:
https://github.com/w3c/browser-specs/pull/286/files